pwa-devdocs/scripts/lint/index.js

```diff
@@ -7,11 +7,18 @@ const config = {
     basePath: path.join(__dirname, '../../src')
 };
 
-// If a single file is specified, only run tests on that file
+// If a file or directory is specified, only run tests on that file or directory
 const filepath = process.argv[2];
 if (filepath) {
     const fullPath = path.join(__dirname, '..', '..', filepath);
-    lintFile(fullPath);
+
+    fs.stat(fullPath, (error, stats) => {
+        if (stats.isFile()) {
+            lintFile(fullPath);
+        } else if (stats.isDirectory()) {
+            lintDirectory(fullPath);
+        }
```

Comment:
The functionality has changed here if `filepath` is invalid (not a file or a directory).

Previously it would have called `lintFile` no matter what (which would probably cause some errors to appear in the console), but now it takes no action.

Just pointing it out in case that is not the desired behavior. You may or may not want to add a default `else` case that tells the user their `filepath` is invalid.